posts/2022-05-12-stacktrace-log-22006-beta.adoc

```diff
@@ -0,0 +1,170 @@
+----
+
+This new functionality is enabled by configuring either a bootstrap property, an environment variable, or through the `server.xml` file. If configuration is present in all these options, the configuration precedence is the `server.xml` > environment variable > bootstrap property.
+
+*Configuration:*
+
+- `bootstrap.properties`: 
+[source]
+----
+`com.ibm.ws.logging.stackJoin=true
+----
+- `server.env`: 
+[source]
+----
+WLP_LOGGING_STACK_JOIN=true/false
```

Comment:
why is `true/false` specified in 2/3 of these examples if that's not an actual value and the default is false? I suppose it could be to demonstrate that one config source could override another in some scenarios? But seems a little muddled without more explanation

Reply:
I agree, I restructured this section with the intent to replace them with just `true` to show how to enable the functionality, but apparently only changed it for the first one.  Going to make that change now